apps/frontend/src/components/TimeCardPage/EvaluationModal.tsx

```diff
@@ -0,0 +1,182 @@
+import {
+  Stack,
+  Button,
+  Modal,
+  ModalContent,
+  ModalHeader,
+  ModalCloseButton,
+  ModalBody,
+  FormLabel,
+  RadioGroup,
+  Radio,
+  Input,
+} from "@chakra-ui/react";
+import React from "react";
+
+import { useState, useEffect } from "react";
+import { CommentSchema } from "src/schemas/RowSchema";
+import { createNewComment } from "./utils";
+import { UserSchema } from "../../schemas/UserSchema";
+import { CommentType } from "./types";
+
+export function EvaluationModal({
+  isOpen,
+  onClose,
+  onCommentSubmit,
+  onCommentDelete,
+  commentToEdit,
+}: {
+  isOpen: boolean;
+  onClose: () => void;
+  onCommentSubmit: (comment: CommentSchema) => void;
+  onCommentDelete: (comment: CommentSchema) => void;
+  commentToEdit: CommentSchema | null;
+}) {
+  const [editedComment, setEditedComment] = useState<string>(
```

Comment:
For any state variables that aren't super explicit, would be great to have a single line comment describing what their purpose is

Reply:
I think editedComment is alright to leave as is